src/@next/Menu/Menu.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import { ListContainer, StyledMenu, TitleContainer } from './MenuStyle';
+
+interface Option {
+  active?: boolean;
+  disabled?: boolean;
+  id?: string;
+  label: React.ReactNode;
+  value: string;
+}
+export interface MenuProps {
+  id?: string;
+  onChange?: (selected: string[]) => void;
+  options?: Option[];
+  selected?: string[];
+  title?: string;
+}
+
+export const Menu = ({ id, onChange, options, selected, title }: MenuProps) => {
+  const handleClick = useCallback(
+    (value: string) => {
```

Comment:
why do we need useCallback here? 🤔

Reply:
let me remove it as it's not needed here